tests/conftest.py

```diff
@@ -59,17 +63,43 @@ def testdata_cache_path() -> str:
     return TESTDATA_CACHE_PATH
 
 
+@pytest.fixture(
+    params=[
+        PYTHON_VERSION,
+        pytest.param(
+            MIN_PYTHON_VERSION,
+            marks=pytest.mark.skipif(
+                MIN_PYTHON_VERSION >= PYTHON_VERSION,
+                reason="requires lower Python version",
```

Comment:
I am not sure what does the reason mean here? for example if the current system python version is `3.8` then the parameter `(3, 9)` is not going to be used? and therefore the `reason` is probably misleading. Could you clarify please?

Reply:
My original thought was to test three cases of a worker environment with the same, lower, or higher Python version than the main process, so `MIN_PYTHON_VERSION` is only tested when it's lower than the main Python for example. This is updated now to just test all python versions between `MIN_PYTHON_VERSION` and `MAX_PYTHON_VERSION`.